DependencyInjection/Configuration.php

```diff
@@ -85,6 +85,9 @@ public function getConfigTreeBuilder()
                         ->scalarNode('enable_sibling_provider')->defaultValue(false)->end()
                     ->end()
                 ->end()
+                ->arrayNode('content_listener')
+                    ->canBeDisabled()
```

Comment:
@dbu I think the way this is handled by FOSUserBundle where you can override the service id for the user manager is that the default service's id is like cmf_seo.content_listener_default and the dependency injection extension sets up cmf_seo.content_listener as an alias based on the configuration